src/Components/Forms/RandomForm.jsx

```diff
@@ -0,0 +1,15 @@
+
+
+export default function RandomForm({numOfRandom, setRandomQuotes, setNum}) {
+    return (
+        <form onSubmit={setRandomQuotes}>
+             <select value={Number(numOfRandom)} onChange={(e) => setNum(e.target.value)}>
+                <option value='1' >1</option>
+                <option value='5' >5</option>
+                <option value='10' >10</option>
+                <option value='25' >25</option>
```

Comment:
this is a little WET - it would be nice to map through an array of option values that's passed in as a prop so that they can be easily changed in the future